rcl/Doxyfile

```diff
@@ -19,8 +19,8 @@ PREDEFINED             += RCL_PUBLIC=
 PREDEFINED             += RCL_WARN_UNUSED=
 
 # Uncomment to generate internal documentation.
-ENABLED_SECTIONS       = INTERNAL
-INPUT                  += ./src/rcl/arguments.c
+# ENABLED_SECTIONS       = INTERNAL
+# INPUT                  += ./src/rcl/arguments.c
```

Comment:
Let's just remove these two lines; there is no reason for them.

Reply:
Done in https://github.com/ros2/rcl/pull/932/commits/beda62e00bd1e8c320ff67171146c11df61b2c27.